assets/js/blocks/reviews/edit-utils.js

```diff
@@ -1,9 +1,9 @@
 /**
  * External dependencies
  */
-import { __, sprintf } from '@wordpress/i18n';
-import { Fragment, RawHTML } from '@wordpress/element';
-import { escapeHTML } from '@wordpress/escape-html';
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { __experimentalCreateInterpolateElement } from 'wordpress-element';
```

Comment:
Would make it sense to stop using `<Fragment>` and use `<>` instead? We currently have three different packages where `Fragment` can be imported from (`@wordpress/element`, `wordpress-element` and `react`), so the less we need to import from them the fewer possible bugs we might introduce.

I see Gutenberg is a bit inconsistent with this (sometimes it uses `<>` and sometimes `<Fragment>`), but we could enforce using `<>` everywhere.

I think @senadir also suggested something like that in a previous PR, so cc'ing him.

Reply:
I'm for using `<>`.  I think our linting tools and babel pragama's are up to date so that it's handled correctly? Assuming they are, it'd probably be good to standardize around that usage.

Reply:
TIL

Reply:
Issue created: #1761.